### PR TITLE
Add base_fee_per_gas, priority_fee_per_gas and chain_id to neon_trans…

### PR DESCRIPTION
**Pull request by spylogsster**
Resolves https://neonlabs.atlassian.net/browse/NDEV-2954

### CODE REVIEW

File: db/scheme.sql [hunk 1]
```diff
@@ -169,9 +169,19 @@ CREATE TABLE IF NOT EXISTS neon_transactions (
     r TEXT,
     s TEXT,
 
+    base_fee_per_gas TEXT,
+    chain_id INT,
```
Comment (afalaleev) [inline db/scheme.sql:173]: DEFAULT 0

File: indexer/db/neon_tx_db.py [hunk 2]
```diff
@@ -220,8 +220,22 @@ class _Record:
...
+
     @classmethod
     def from_tx(cls, neon_tx: NeonTxModel, neon_rcpt: NeonTxReceiptModel) -> Self:
+        base_fee_per_gas = None
+        priority_fee_per_gas = None
+        tx_chain_id = None
+        # Supported only for new transaction type
+        if neon_tx.tx_type == 2:
+            base_fee_per_gas = neon_tx.max_fee_per_gas - neon_tx.max_priority_fee_per_gas
+            priority_fee_per_gas = neon_tx.max_priority_fee_per_gas/base_fee_per_gas if base_fee_per_gas > 0 else 0
```
Comment (afalaleev) [inline indexer/db/neon_tx_db.py:235]: IMHO, it is better to store the values as they were defined by an user.
And if it is required, convert them on the API level.

File: indexer/db/neon_tx_db.py [hunk 3]
```diff
@@ -247,6 +261,9 @@ def from_tx(cls, neon_tx: NeonTxModel, neon_rcpt: NeonTxReceiptModel) -> Self:
             is_canceled=neon_rcpt.is_canceled,
             is_completed=neon_rcpt.is_completed,
             logs=cls._encode_event_list(neon_rcpt),
+            base_fee_per_gas=base_fee_per_gas,
+            chain_id=tx_chain_id,
+            priority_fee_per_gas=priority_fee_per_gas,
```
Comment (afalaleev) [inline indexer/db/neon_tx_db.py:266]: _RecordWithBlock.to_meta() is used for loading a NeonTx from the db.
Please implement the logic

**Comment by evgenyzdanovich** [comment]
I adapted this change into the branch and resolved the comments. let's continue the discussion in the main branch.

**Pull request closed without merging**